### PR TITLE
1.12.x

### PR DESCRIPTION
**Pull request by cloudhakasekumo403**
Tiny fix recommendation for docs/gettingstarted/index.md
1 setupDecompWorkspace problem
2 runClient problem

### CODE REVIEW

**Comment by Jonathing** [comment]
1.12, but it's documentation so I'm not sure what to do with this. Leaving the decision up to @ChampionAsh5357

**Comment by ChampionAsh5357** [comment]
These comments are not really documentation. Based on the comments made, it seems as though you have an error in your own buildscript. We have verified in the past (by which I mean within the last six months) that the gradle tasks work with some modification to the buildscript.

I will keep this open if you want to provide documentation towards the changes necessary to have it run. However, the current state requires more details and an actual understanding of the what could potentially be causing your issue.

**Comment by cloudhakasekumo403** [comment]
According to https://github.com/MinecraftForge/Documentation/issues/520#issuecomment-1782202746, `setupDecompWorkspace` is deleted. And I will modify the pull request.

File: docs/gettingstarted/index.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ From Zero to Modding
 5. Choose your IDE: Forge explicitly supports developing with Eclipse or IntelliJ environments, but any environment, from Netbeans to vi/emacs, can be made to work.
     * For Eclipse, you should run `gradlew eclipse` - this will download some more artifacts for building eclipse projects and then place the eclipse project artifacts in your current directory.
     * For IntelliJ, simply import the build.gradle file.
+
+!!! note    
+
+    `setupDecompWorkspace` is deleted (tested in `1.12.2 - 14.23.5.2860`) and you can skip step 4.
+
```
Comment (ChampionAsh5357) [inline docs/gettingstarted/index.md]: This is not how you write docs. You can just copy the section text from one of the other branches which uses FG3 (1.13-1.15). We do not need to special case for people using FG2 here and keep the old setup stuff.

File: docs/gettingstarted/index.md [hunk 2]
```diff
@@ -33,6 +38,7 @@ From Zero to Modding
     assign more RAM into gradle by adding `org.gradle.jvmargs=-Xmx2G` into the file `~/.gradle/gradle.properties` (create file if doesn't exist). The `~` sign means it's a user's [home directory][]    .
 
 
+
```
Comment (ChampionAsh5357) [inline docs/gettingstarted/index.md]: Redundant line.

File: docs/gettingstarted/index.md [hunk 3]
```diff
@@ -82,3 +88,9 @@ Building and Testing Your Mod
 !!! note
 
     It is always advisable to test your mod in a dedicated server environment if it is intended to run there.
+
+
+!!! note
+
+    For `forge-1.12.2-14.23.5.2860-mdk` version, it looks like `gradlew runClient` will ignore all assets under `src/main/resourcs`. (BUG?)   
+
```
Comment (ChampionAsh5357) [inline docs/gettingstarted/index.md]: Once again, no idea what this is about. I've verified it works fine, barring a few changes such that the build script looks like the 1.15 version.

**Review by ChampionAsh5357 — CHANGES_REQUESTED** [review]
Please rewrite the docs instead of leaving notes.

**Comment by cloudhakasekumo403** [comment]
Thank you for helping! I will rewrite docs.

File: docs/gettingstarted/index.md (hunk 3 above)
Comment (cloudhakasekumo403) [reply docs/gettingstarted/index.md]: Temporarily deleted the note. I feel like this problem should discussed using github issue once.

File: docs/gettingstarted/index.md (hunk 2 above)
Comment (cloudhakasekumo403) [reply docs/gettingstarted/index.md]: Deleted the line.

File: docs/gettingstarted/index.md (hunk 1 above)
Comment (cloudhakasekumo403) [reply docs/gettingstarted/index.md]: I copied the section from 1.13.x branch, does it look like ok?

**Review by ChampionAsh5357 — CHANGES_REQUESTED** [review]
Honestly, just copy the text from the 1.13-1.15 page for this version.  There's nothing version specific, so that should solve any issues I have with this.

Apologies, I thought I left a message on this earlier.

**Comment by cloudhakasekumo403** [comment]
I copied `From Zero to Modding` section from 1.13.x. Those files look almost same.

```
$ curl https://raw.githubusercontent.com/kumohakasemk9/forgedocs-fork/1.12.x/docs/gettingstarted/index.md > file1
$ curl https://raw.githubusercontent.com/MinecraftForge/Documentation/1.13.x/docs/gettingstarted/index.md > file2
$ diff file1 file2
```
```
20,21d19
< [files]: https://files.minecraftforge.net "Forge Files distribution site"
< 
50c48
< 3. You can also run a dedicated server using the server run config, or `gradlew runServer`. This will launch the Minecraft server with its GUI.
---
> 3. You can also run a dedicated server using the server run config, or `gradlew runServer`. This will launch the Minecraft server with it's GUI.
54a53
```

**Comment by ChampionAsh5357** [comment]
Ah, I was looking at the 1.15 one. Didn't realize that the 1.13 one was missing some info. It doesn't matter which one since they all use FG3, but copy the 1.15 edition please.

Apologies for the misunderstanding

**Comment by cloudhakasekumo403** [comment]
Thank you for review. I copied docs from `MinecraftForge/Documentation/edit/1.15.x/docs/gettingstarted/index.md`. But I am confused that I should replace only `From 0 to modding` or entire file. However, 1.15.x docs looked so nice and I did entire file replacement.

**Review by ChampionAsh5357 — APPROVED** [review]
Yep, looks good

**Comment by cloudhakasekumo403** [comment]
Thank you very much.

**Pull request merged**